drivers/ethernet/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ zephyr_sources_ifdef(CONFIG_ETH_ENC28J60	eth_enc28j60.c)
 zephyr_sources_ifdef(CONFIG_ETH_MCUX		eth_mcux.c)
```

Comment:
You need to add proper commit message + signed-off-by to both commits.

Reply:
I extended the commit messages and added signed-off tags.